packages/suite/src/reducers/suite/__fixtures__/deviceReducer.ts

```diff
@@ -530,6 +530,33 @@ const changed = [
         ],
         result: [],
     },
+    {
+        description: `features are not overridden when device is locked`,
+        initialState: [
+            getSuiteDevice(
+                // Reducer doesn't try to merge non-connected devices.
+                // Set `connected` to `true` to overcome that.
+                { connected: true },
+                { safety_checks: 'Strict', unlocked: true },
+            ),
+        ],
+        actions: [
+            {
+                type: DEVICE.CHANGED,
+                payload: getConnectDevice(undefined, {
+                    unlocked: false,
+                    safety_checks: 'PromptAlways',
```

Comment:
```suggestion
                    safety_checks: null,
```
tested case should reflect reality

Reply:
Edited, see [the fixup](https://github.com/trezor/trezor-suite/pull/3590/commits/814d5899fb53d06ca7e1537d13b3970c5e2e663e).